Lib/uuid.py

```diff
@@ -475,74 +475,100 @@ def _netbios_getnode():
             continue
         return int.from_bytes(bytes, 'big')
 
+def _is_darwin_greater_than_version(major_version_number):
+    """Checks if the operating system is Darwin newer than
+    the specified major version number."""
+    if sys.platform != "darwin":
+        return False
+    return (int(os.uname().release.split(".")[0]) >= major_version_number)
+
+# The uuid_generate_* functions are broken on MacOS X 10.5, as noted
+# in issue #8621 the function generates the same sequence of values
+# in the parent process and all children created using fork (unless
+# those children use exec as well).
+#
+# Assume that the uuid_generate functions are broken from 10.5 onward,
+# the test can be adjusted when a later version is fixed.
+_no_uuid_generate_time_lookup = (_is_darwin_greater_than_version(9) is True)
```

Comment:
This is redundant: ``(true expression) is True``

Reply:
I wanted to be explicit that this is a boolean value. Think that's much easier to read.

Reply:
especially considering that nothing in this module is explicitly typed.

Reply:
But it’s not the python way.  `_dont_do_something = _detect_thing()` is legible.

Reply:
I'm happy to adopt suggestions for better naming. After thinking back and forward (pairing with @igalic) we ended with this solution. Personally I'd rather make an exception to the naming pattern for boolean values than complicating the logic here.

Reply:
i didn't notice that the function is actually called `_is_this_thing()`… fixed according to @merwok's comment.